lib/gateway/contentful_gateway.rb

```diff
@@ -153,12 +153,28 @@ def parse_content(content)
       create_container(content)
     when 'imageLinkWithDescription'
       create_image_link_with_description(content)
+    when 'download'
+      create_download(content)
     else
       @logger.warn("Content #{content.sys[:content_type].id} not supported")
       nil
     end
   end
 
+  def create_download(content)
+    puts content.download_resource.file.url
+    {
+      type: :download,
+      data: {
+        name: content.name,
+        thumbnail_image_src: content.thumbnail_image.url,
+        download_resource_link: content.download_resource.url,
```

Comment:
```suggestion
        resource_link: content.resource.url,
```

I think `download` is unnecessary as it's a type field for a `download` type. Same thoughts for `download_resource_type` and `download_link_text`.

Reply:
Makes sense. Ive changed it in the gateway, component renderer and on contentful 👍